pre_commit_hooks/check_case_conflict.py

```diff
@@ -1,4 +1,5 @@
 import argparse
+import pathlib
```

Comment:
I would prefer if you don't use pathlib, thanks!

Reply:
is there any specific reason?

Reply:
you can accomplish the same without it, and I don't like pathlib